vizro-core/mkdocs.yml

```diff
@@ -1,5 +1,5 @@
 site_name: Vizro
-site_url: https://mckinsey.github.io/vizro/
+site_url: https://vizro.readthedocs.io/en/stable/
```

Comment:
Do we need something similar for the docs on vizro-ai?

Reply:
Good point! I forgot those docs (as usual). Let me add it.

Reply:
No worries, I always forget about them too 😆